Sources/DotNetGraph.Tests/Indentation/IndentedGraphTests.cs

```diff
@@ -141,5 +141,39 @@ public void IndentedNodeWithAttributes()
 
             Check.That(compiled).HasSameValueAs("graph TestGraph { \n\tTestNode[color=\"#FF0000\",style=bold]; \n}");
         }
+
+        [Fact]
+        public void DotGraph_WhenNodesSubGraphsEdgesAndCustomLinesProvided_ThenCompilationIsFormattedCorrectly()
+        {
+            var graph = new DotGraph("TestGraph", true)
+                .AddLine("rankdir = TB;")
+                .NewSubGraph("cluster_0", s =>
+                {
+                    s.Label = "Test Sub Graph";
+                    s.NewNode("A")
+                        .NewNode("B")
+                        .AddLine("{rank = same; A; X;}");
+                })
+                .NewEdge("A", "B");
+
+            var compiled = graph.Compile(true);
+
+            _output.WriteLine(compiled);
+
+            // digraph TestGraph { 
+            // 	rankdir = TB; 
+            // 	subgraph cluster_0 { 
+            // 		label="Test Sub Graph";
+            // 		A; 
+            // 		B; 
+            // 		{rank = same; A; X;} 
+            // 	} 
+            // 	A -> B; 
+            // }
+
+            var expected2 = "digraph TestGraph { \n\trankdir = TB; \n\tsubgraph cluster_0 { \n\t\tlabel=\"Test Sub Graph\"; \n\t\tA; \n\t\tB; \n\t\t{rank = same; A; X;} \n\t} \n\tA -> B; \n}";
```

Comment:
Small detail but can be renamed to `expected`

Reply:
Fixed